include/rs_utils.h

```diff
@@ -78,6 +78,8 @@ char* rs_program_version();
 
 unsigned int rs_newsboat_version_major();
 
+int rs_mkdir_parents(const char* path, const unsigned int mode);
```

Comment:
It's better to use an integer of fixed width, `std::uint32_t` in this case, because Rust expects precisely that size and signedness.

I notice we make the same mistake of using `int` all over our FFI bindings. I'll open an issue about this.

Reply:
Done. I've also pushed your comments into the commit bodies.